README.md

```diff
@@ -744,10 +744,12 @@ _See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems_
 
 #### Single-click/drag-n-drop upload
 
+- [BlackUpload](https://github.com/FarisCode511/BlackUpload) - An anonymous file uploader script with multi-file upload support. `MIT` `PHP`
 - [BoZoN](https://github.com/broncowdd/BoZoN) - Minimalist Drag and drop file sharing app. `AGPL-3.0` `PHP`
 - [Coquelicot](https://coquelicot.potager.org/) - Coquelicot is a “one-click” file sharing web application with a focus on protecting users’ privacy. ([Source Code](https://coquelicot.potager.org/gitweb/?p=coquelicot.git)) `AGPL-3.0` `Ruby`
 - [elixire](https://elixi.re) - Simple yet advanced screenshot uploading and link shortening service. ([Source Code](https://gitlab.com/elixire/elixire), [Clients](https://gitlab.com/elixire/elixiremanager)) `AGPL-3.0` `Python`
 - [fibridge](https://github.com/anderspitman/fibridge-proxy-rs) - Stream huge files out of your browser without having to upload. `MIT` `Rust`
+- [FireShare](https://auspham.dev/FireShare/) - A full-stack, pub-sub, real-time secure file sharing system. Using NodeJS, ReactJS, SocketIO, MongoDB. Chai, WebdriverIO for testing. ([Source Code](https://github.com/rockmanvnx6/FireShare)) `Nodejs` `MIT`
```

Comment:
Please remove the technical stack description `Using NodeJS, ReactJS, SocketIO, MongoDB. Chai, WebdriverIO for testing.`

Reply:
deleted. ;)